docs/release_notes/8_8.md

```diff
@@ -53,80 +53,83 @@ Some highlights and new features from this release are detailed next.
+* `libreswan` packages have been rebased to version 4.9
 * You can now filter the RPM database with `fapolicyd` packages.
-* `openscap` packages were rebased to version 1.3.7.
+* `openscap` packages were rebased to version 1.3.7
 * OpenSSL driver can now use certificate chains in Rsyslog.
 * **systemd-socket-proxyd** service now runs in its own SELinux domain due to an update to the SELinux policy.
 
-### Dynamic programming languages, web and database servers
+Security-related highlights in the latest Rocky Linux 8.8 release are listed below. For a complete list of security related changes, see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-security).
 
-For a detailed list of the changes in this category see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-dynamic-programming-languages-web-and-database-servers).
+### Dynamic programming languages, web and database servers
 
-More recent versions of the following Application Streams are now available:
+The latest version of the following application module stream is now available:
 
-* nginx 1.22
-* PostgreSQL 15
-* Jakarta XML Binding (`jaxb`) 4
-* Swig 4.1
+* `nginx:1.22`
+* `postgresql:15`
 
-The following applications were added:
+The following packages have been added:
 
-* Python 3.11
-* Tomcat 9
+* `python3.11`. Note that, unlike the previous versions, Python 3.11 is distributed as standard RPM packages instead of a module.
+* `tomcat`. The version of the package is 9
 
-The following components have been added or upgraded:
+The following components have been upgraded:
 
-* Git to version 2.39.1:
+* `git` packages rebased to version 2.39.1
 
-    * Commit signing with SSH keys supported is supported now
+* `git-lfs` packages rebased to version 3.2.0
 
-* Git LFS to version 3.2.0
+For a detailed list of the changes in this category see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-dynamic-programming-languages-web-and-database-servers).
 
 ### Compilers and Development Tools
 
-For details on the changes in this category see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-compilers-and-development-tools).
+The latest version of the following application module stream is now available:
+
+* `jaxb:4`
+* `Swig:4.1`
 
-#### Updated Compiler Toolsets
+The following compilers or development tools have been updated:
 
-* GCC Toolset 12
-* LLVM Toolset 15.0.7
-* Rust Toolset 1.66
-* Go Toolset 1.19.4
+* `gcc-toolset-12`
+* `rust-toolset` packages rebased to version 1.66.1
+* `llvm-toolset` packages rebased to version 15.0.7
+* `go-toolset` packages rebased to version 1.19.6
+
```

Comment:
I don't remember if this was my wording in the first place, but as well here, I don't think writing the package name in the release notes is the correct way, because it's rather a set of packages than just one package.

Reply:
> I don't remember if this was my wording in the first place, but as well here, I don't think writing the package name in the release notes is the correct way, because it's rather a set of packages than just one package.

`dnf -y install rust-toolset llvm-toolset go-toolset`